_pages/algorithm-development/comparing-algorithms.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: article
+title:  "Evaluations"
+excerpt: "This guide will walk you through the steps to implement your own evaluation."
+categories: algorithm-basics
+tags: [algo-dev]
+show_related: true
+image:
+    teaser: /icons/algo.svg
+permalink: /algorithm-development/algorithm-basics/evaluations/
+redirect_from:
+  - /algorithm-development/evaluations/
+---
+
+So you've been playing around with Algorithmia for a while, and you may have noticed a variety of algorithms that tackle the same problem, in different ways. The [Algorithmia Evaluations](https://algorithmia.com/evaluations) tool lets you compare algorithms in a bunch of different ways. 
+This guide will walk you through the steps to implement your own evaluation.
+
+**Note:** Things you should have before we begin:
+- A working knowledge of [algorithm development on Algorithmia](https://algorithmia.com/developers/algorithm-development/).
+- A type of algorithm you want to evaluate (image classification, character recognition, etc).
+- Some criteria you want to compare your algorithms with.
+
+Got all that? Great! Lets dive in.
+
+## Step 1: Find your evaluation data
+ ![](https://i.imgur.com/i8Mxzd9.png)
+- If your evaluation requires labelled data comparison, make sure you have the data available and formatted for use in the [data API](https://algorithmia.com/data).
+
+## Step 2: Create an Evaluation (evaluator) algorithm.
+- An evaluator algorithm is special type of algorithm that compares similar algorithms based on some kind of criteria.
```

Comment:
"An evaluator algorithm is special" should be "An evaluator algorithm is **a** special"

Reply:
thanks! Changed it.